mypyc/primitives/int_ops.py

```diff
@@ -139,3 +146,7 @@ def int_unary_op(name: str, c_function_name: str) -> CFunctionDescription:
 
 
 int_neg_op = int_unary_op('-', 'CPyTagged_Negate')
+
+int_logical_op_mapping = {
+    '==': (BinaryIntOp.EQ, int_equal)
+}  # type: Dict[str, Tuple[int, CFunctionDescription]]
```

Comment:
Add comment saying that these are not complete implementations, but assume that one of operands is a long integer.

Reply:
Actually there's no such assumption I think, the first item of the tuple provides op variant for the case when lhs of the operands is short, therefore not assume at least one of them is long.

Reply:
Ah yeah, you are correct.